recipes/rpy2/build.sh

```diff
@@ -10,7 +10,6 @@ if [ "$(uname)" == "Darwin" ]; then
     export CFLAGS="${CFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
     export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
     export CXXFLAGS="${CXXFLAGS} -stdlib=libc++"
-    export LDFLAGS="${LDFLAGS} -headerpad_max_install_names"
```

Comment:
On second thought, it is needed if a recipe build two shared libraries and one is linking the other